collections/_tutorials/2022-09-12-tutorial-cvae.md

```diff
@@ -0,0 +1,296 @@
+
+$$\left(f^*,g^*,h^*,k^*,l^*\right) = \underset{(f,g,h,k,l)}{\arg\min} \,\,\, \left( \mathbb{E}_{z\sim q(z \vert x,y)} [CE\left( y,f\left(x,z\right)\right)] + D_{KL}(\underbrace{q(z \vert x,y)}_{g,h}\parallel \underbrace{p(z \vert x)}_{k,l}) \right)$$
+
+&nbsp;
+
+## Various scenarios
+
+There are different ways to leverage the cVAE formalism depending on the prior modeling $$p(z \vert x)$$ and the content of the $$y$$ references.
+
+### Prior modeling
+
+* The prior $$p(z \vert x)$$ outputs a latent variable $$z$$ depending on the input $$x$$. This means that the corresponding latent space will be structured according to a varying input $$x$$ as illustrated in the figure below.
+
+![](/collections/images/cvae/cvae_prior_depending_on_x.jpg)
+
+
+* Several works in the literature propose to relax this constraint to make the latent variables statistically independent of input variables, i.e. $$p(z \vert x) = p(z)$$ with $$z \sim \mathcal{N}\left(0,I\right)$$. This implies that the latent space is forced to be centered at the origin with unit variance, which makes the posterior modeling strategy close to that used in standard VAE, as illustrated in the figure below.
+
+![](/collections/images/cvae/cvae_prior_no_depending_on_x.jpg)
+
+### Nature of the references
+
+Depending on the type of reference available, the value of conditional VAE can be different.
+
+* If there exists only one reference for a given input, the interest of the conditional VAE resides in the mixing of the input $$x$$ data with the corresponding $$y$$ in the latent space through the modeling of $$p(z \vert x,y)$$. This can be viewed as a regularisation process that "efficiently" integrates reference information during inference thanks to the dedicated latent space and the mapping $$p(y \vert x,z)$$. 
+
+> In the context of segmentation, modeling $$p(z \vert x,y)$$ can be seen as an "efficient" way to integrate shape prior into the latent space. 
+
+&nbsp;
+
+* If there exists several references for a given input, which is the case when we want to model inter/intra-expert variability, the interest of the conditional VAE resides in its capacity to model the reference variability in the latent space through the modeling of $$p(z \vert x,y)$$. This way, a single input corresponds to several latent variables that are located in the same region of the space, as illustrated in the figure below.
+
+![](/collections/images/cvae/cvae_prior_depending_on_x_multiple_y.jpg)
+
+During inference, the latent space modeled by the prior is sampled several times to generate multiple plausible references $$\hat{y}_i$$ from a given input $$x$$, taking into account the learned variability as shown in the figure below.
+
+![](/collections/images/cvae/cvae_prior_depending_on_x_multiple_y_inference.jpg)
+
+> In the context of segmentation, this approach is useful for learning inter-expert variability.
+
+&nbsp;
+
+## Simple example
```

Comment:
c'est probablement correct, mais ce n'est pas intuitif de représenter la data x comme étant une étiquette de classe et la "référence" y comme une image.

Reply:
I agree and we discussed this point together today :)